mappings/net/minecraft/client/gui/screen/world/WorldListWidget.mapping

```diff
@@ -1,38 +1,57 @@
 CLASS net/minecraft/unmapped/C_wwgrvynj net/minecraft/client/gui/screen/world/WorldListWidget
 	FIELD f_ahgrbbnb DATE_FORMAT Ljava/text/DateFormat;
 	FIELD f_biwkcpnf SNAPSHOT_FIRST_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_bnkwjzpz listFuture Ljava/util/concurrent/CompletableFuture;
 	FIELD f_bvfzqfwz CONVERSION_TOOLTIP_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_giqqdqgq levels Ljava/util/List;
 	FIELD f_gmbxrzpw FROM_NEWER_VERSION_FIRST_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_gmmkqxli SNAPSHOT_SECOND_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_iwchirdh loadingEntry Lnet/minecraft/unmapped/C_wwgrvynj$C_chlqbstq;
+	FIELD f_lejfmpml query Ljava/lang/String;
 	FIELD f_mbtquola parent Lnet/minecraft/unmapped/C_yrypplpi;
 	FIELD f_neodhkus FROM_NEWER_VERSION_SECOND_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_nhmjctua LOGGER Lorg/slf4j/Logger;
 	FIELD f_rtudrfdo LOCKED_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_rxeuofpy WORLD_SELECTION_LOCATION Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_xjlsplba UNKNOWN_SERVER_LOCATION Lnet/minecraft/unmapped/C_ncpywfca;
-	METHOD m_adqicvjj narrateScreenIfNarrationEnabled ()V
-	METHOD m_bhyykuuh shouldShow (Ljava/lang/String;Lnet/minecraft/unmapped/C_wfuumzdp;)Z
-		ARG 1 search
+	METHOD <init> (Lnet/minecraft/unmapped/C_yrypplpi;Lnet/minecraft/unmapped/C_ayfeobid;IIIIILjava/lang/String;Lnet/minecraft/unmapped/C_wwgrvynj;)V
+		ARG 1 parent
+		ARG 2 client
+		ARG 3 screenWidth
+		ARG 4 screenHeight
+		ARG 5 top
+		ARG 6 bottom
+		ARG 7 itemHeight
+		ARG 8 query
+		ARG 9 previousList
+	METHOD m_adqicvjj narrateScreen ()V
+	METHOD m_bhyykuuh worldNameMatches (Ljava/lang/String;Lnet/minecraft/unmapped/C_wfuumzdp;)Z
```

Comment:
I'm not overly clear on what this method does, the previous name `shouldShow`, is wildly different to the new one and doesn't really help either. Could you just check this one please?

Reply:
they filter worlds by name